lang/system/pl.xml

```diff
@@ -357,25 +829,30 @@ System Vita3K jest gotowy!</completed_setup>
 		<content_manager>Menedźer zawartości</content_manager>
 	</sys_apps_title>
 
-	<trophy_collection>
+	<trophy_collection name="Kolekcja trofeów">
```

Comment:
revert it, this line not used the "name".

Reply:
Done, my bad.